src/tutorials/dropbox.md

```diff
@@ -0,0 +1,69 @@
+    tar xzvf archive.tar && rm archive.tar
+```
+
+The build hook runs when your application's codebase is being assembled for deployment, so the filesystem is still writable.  This means that the above command will create a `~/.dropbox-dist` directory in the home directory of your application, aka "the application root".  Starting the daemon is _almost_ as simple as running `~/.dropbox-dist/dropboxd` when you `ssh` into your application's container.
+
+## Creating the necessary writable directories
+
+You'll need to add two writable/mounted directories that Dropbox expects to be there and to be writable.  The main one is `~/Dropbox`, which contains **your entire Dropbox**.  The other is a metadata directory at `~/.dropbox`.  This is typically modestly sized, on the order of less than 100MB.  
+
+Because Dropbox files are mirrored on your writable disk mount, your application container's disk size needs to be large enough to accommodate all of your Dropbox files as well as any user-uploaded files and your own code. You should also include sufficient extra space to allow for growth over time.
+
+```yaml
+# for a 1G total Dropbox size, 
+# leaving another 1G for the application
+disk: 2048
+# the needed mounts for the Dropbox daemon
+mounts:
+  "Dropbox": "shared:files/dropbox"
+  ".dropbox": "shared:files/dropbox-meta"
+```
+
+**Make sure you've `git push`ed these configuration changes to Platform.sh if you haven't already.**
+
+## Starting the daemon
+
+You'll eventually be building this step in to your [deploy hook](/configuration/app-containers.html#hooks), but first you'll need to start it manually so that you can login to Dropbox from your application's container.  `ssh` into your application's container, either by grabbing the URL from the "access site" link in your project's admin UI or by running `platform ssh` if you've installed the [Platform CLI](https://docs.platform.sh/overview/cli.html) (recommended).
+
+You should see a `~/.dropbox-dist` directory as a result of your build hook.  Run this - `~/.dropbox-dist/dropboxd` and you'll be presented with some text in `stdout` that asks you to visit a tokenized Dropbox URL that will authenticate your new Dropbox client against your account and store the results in the `~/.dropbox` directory.
+
+After authenticating, you'll notice that the client just hangs there because by default it runs in the foreground.  Press control-c to kill the process.  Add this startup command to your deploy hook, modified to run in the background.
+
+Edit your `hooks` section to look like this -
+
+```yaml
+hooks:
+  build: |
+    wget "https://www.dropbox.com/download?plat=lnx.x86_64" -O archive.tar
+    tar xzvf archive.tar && rm archive.tar
+  deploy: |
+    # starts the process as a daemon and sends all
+    # output to /dev/null.  Obviously, feel free to
+    # direct this output to a logfile(s) if you wish.
+    nohup ~/.dropbox-dist/dropboxd >/dev/null 2>&1 &
```

Comment:
It _could_ be in `web.commands.start`, I don't understand why that would be a benefit to put it there though vs the deploy hook.